server.js

```diff
@@ -21,8 +21,8 @@
 const http = require('http');
 const { MongoClient } = require('mongodb');

```

Comment:
Consider using a naming convention for constants, such as using all uppercase letters for the variable name 'HOSTNAME' to indicate that it is a constant value